package.json

```diff
@@ -21,8 +21,7 @@
     "test:coverage": "npm run build && npm test -- --no-watchman --coverage",
     "test:ci": "npm run test -- --no-watchman --maxWorkers=10",
     "test:coverage:ci": "npm run test:ci -- --coverage && codecov",
-    "release:semantic": "lerna-semantic-release pre && lerna-semantic-release post && lerna-semantic-release perform",
-    "release:manual": "./scripts/publish-manually",
+    "release": "npm run build && lerna publish --registry=https://registry.npmjs.org/ --conventional-commits && npm run docs:publish",
```

Comment:
I don't think the `docs:publish` should be included in the release. We can actually always run this on master.

Reply:
1. Releasing should always happen from `master` only with the workflow
   - Feature branch gets merged
   - Pull master and release
2. It won't be included in the release as lerna publish will commit a "Publish" commit

Reply:
I mean that we should build the docs as a continuous step on master, not include it when we manually release something.

Reply:
Alright, all good 👍